packages/template-retail-react-app/config/default.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
             isProduction: false
         },
         // if this is true, useOrigin will take x-forwarded-host header into consideration
-        enableXForwardHost: false
+        enableXForwardedHost: false
```

Comment:
I don't know if I like this config option... 🤔 this flag alters the behaviour of a single hook, is it worth having an app level config?

Reply:
I think I'm tiny slightly favoring making this a flag for the hook, not the retail react app config... but i'd love to hear what others think

Reply:
@bendvc  thoughts?

Reply:
Good point @kevinxh. 
What it would be like to create a flag just for hook? How do we expose the config? 🤔

Reply:
If it's the flag for a single hook, how is this hook used? It can be used anywhere in the app, so I'm ok with having an app-level config for now.

Otherwise, then I imagine something like this:
```jsx
const {getOrigin} = useOrigin({fromXForwardedHost: true})
const origin = getOrigin()
```

Reply:
The origin of an app tends to be constant, it'd be weird to allow it to be toggled by a func arg and returns different origins in a same app. 🤔

Reply:
Also, this flag is to create non-breaking. We should try to remove it in the next major version.

Reply:
As discussed, we will use hook-based argument
```
const origin = useOrigin({withXForwardedHeader: true})
```